### PR TITLE
Fix: #105. No trailers.

### PR DESCRIPTION
**Pull request by ioggstream**
## This PR

- no trailers, but no normative language
- add explanation in FAQ, noting that it could be addressed in future specifications

### CODE REVIEW

**Comment by mnot** [comment]
If you're going to preclude trailers, you should more prominently call it a 'header field' rather than just a 'field' (including in the title?)

**Comment by ioggstream** [comment]
I thought about it. Since I was guessing whether a future extension I-D could address this point, I didn't want to be strict: but please let me know. @mnot

File: draft-ietf-httpapi-ratelimit-headers.md [hunk 1]
```diff
@@ -240,7 +240,7 @@ These examples show multiple policies being returned:
    RateLimit-Policy: 10;w=1;burst=1000, 1000;w=3600
 ~~~
 
-This field can be sent in a trailer section.
+This field cannot apper in a trailer section.
```
Comment (darrelmiller) [inline draft-ietf-httpapi-ratelimit-headers.md:243]:
```suggestion
This field cannot appear in a trailer section.
```

File: draft-ietf-httpapi-ratelimit-headers.md [hunk 2]
```diff
@@ -1165,6 +1164,12 @@ RateLimit-Policy: 100;w=60;burst=1000;comment="sliding window", 5000;w=3600;burs
     RateLimit-Remaining field and RateLimit-Reset field for defining the throttling
     behavior.
 
+15. Can I use RateLimit fields in trailers?
+    Servers usually establish whether the request is in-quota before creating a response, so the RateLimit field values should be already available in that moment.
+    Supporting trailers has the only advantage that allows to provide more up-to-date information to the client in case of slow responses.
+    OTOH this complicates client implementation with respect to combining fields and managing intermediaries that drop trailers.
```
Comment (darrelmiller) [inline draft-ietf-httpapi-ratelimit-headers.md:1170]:
```suggestion
    However, this complicates client implementations with respect to combining fields from headers and accounting for intermediaries that drop trailers.
```

**Comment by mnot** [comment]
A future RFC could expand the scope of these to trailers, yes. In _this_ draft, it's important to communicate consistently and clearly, so you should use 'header field' not just 'field'.

**Comment by ioggstream** [comment]
@mnot will do, but I won't probably make it for the next meeting.

**Review by darrelmiller — APPROVED** [review]

**Pull request merged**